docs/endpoints.rst

```diff
@@ -9,15 +9,18 @@ or a Kubernetes cluster, for example.
 The endpoint requires outbound network connectivity. That is, it must be able to connect to
 funcX at `funcx.org <https://api2.funcx.org/v2>`_.
 
-To install the funcX endpoint agent software ::
+The funcX endpoint is available on pypi.org (and thus available via ``pip``).
+However, *we strongly recommend installing the funcX endpoint into an isolated virtual environment*.
+`Pipx <https://pypa.github.io/pipx/installation/>`_ automatically manages
+package-specific virtual environments for command line applications, so install funcX endpoint via::
 
-     $ python3 -m pip install funcx_endpoint
+     $ python3 -m pipx install funcx_endpoint
 
 .. note::
 
-   Please note that the funcx endpoint is only supported on Linux and MacOS.
+   Please note that the funcX endpoint is only supported on Linux and MacOS.
```

Comment:
Given that currently our product _doesn't_ work on macOS (c.f., the semaphore bug, among others), ought we remove the macOS from this sentence?  Perhaps:

    Please note that the funcX endpoint is currently only supported on Linux.